src/styles/App.css

```diff
@@ -31,3 +31,17 @@ main {
 .app {
     width: 100%;
 }
+
+.github-icon {
+    font-size: 1.6rem;
+    color: black;
+}
+
+footer {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #f0f0f0;
```

Comment:
I think we can add this colour to the `index.css` section and use it from there

Reply:
Good idea I have moved it now.